specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json

```diff
@@ -157,7 +157,7 @@
       "post": {
         "summary": "Calculate price for a `ReservationOrder`.",
         "description": "Calculate price for placing a `ReservationOrder`.",
-        "operationId": "ReservationOrder_Calculate",
+        "operationId": "ReservationOrder_CalculatePrice",
```

Comment:
Breaking change here

Reply:
It was not intended to create breaking changes in this PR, we'll roll this back.